---
title: Papers
---

Find our papers at <i class='ai ai-google-scholar ai-lg'></i> [Google Scholar](https://scholar.google.com/citations?user=mjkV0pUAAAAJ) or <i class='ai ai-pubmed ai-lg'></i> [PubMed](https://pubmed.ncbi.nlm.nih.gov/?term=Qiyun+Zhu[Author]).

## Preprints

- Jiang Y, Balaban M, **Zhu Q**, Mirarab S. DEPP: Deep Learning Enables Extending Species Trees using Single Genes. [_BioRxiv_](https://www.biorxiv.org/content/10.1101/2021.01.22.427808v2).
- Yoo BB, Griffiths JA, Thuy-Boun PS, Cantu V, Weldon KC, Challis C, Sweredoski MJ, Chan KY, Thron TM, Sharon G, Moradian A, Humphrey G, **Zhu Q**, Shaffer J, Wolan D, Dorrestein P, Knight R, Gradinaru V, Mazmanian SK. Targeted Activation of Enteric Neurons Shapes the Gut Environment of Mice. [_SSRN_](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3713502).
- Shaffer JP, Nothias LF, Thompson LR, Sanders JG, Salido RA, Couvillion SP, Brejnrod AD, et al. Multi-omics profiling of Earth’s biomes reveals that microbial and metabolite composition are shaped by the environment. [_BioRxiv_](https://www.biorxiv.org/content/10.1101/2021.06.04.446988v3).
- Liu Y, Meric G, Havulinna AS, Teo SM, Ruuskanen M, Sanders J, **Zhu Q**, Tripathi A, Verspoor K, Cheng S, Jain M, Jousilahti P, Vázquez-Baeza Y, Loomba R, Lahti L, Niiranen T, Salomaa V, Knight R, Inouye M. Early prediction of liver disease using conventional risk factors and gut microbiome-augmented gradient boosting. [_MedRxiv_](https://www.medrxiv.org/content/10.1101/2020.06.24.20138933v1).
- Gauglitz JM, Bittremieux W, Williams CL, Weldon KC, Panitchpakdi M, Ottavio FD, Aceves CM, Brown E, Sikora NC, Jarmusch AK, Martino C, Tripathi A, Sayyari E, Shaffer JP, Coras R, Vargas F,Goldasich LD, Schwartz T, Bryant M, Humphrey G, Johnson AJ, Spengler K, Belda-Ferre P, Diaz E, McDonald D, **Zhu Q**, Nguyen DS, Elijah EO, Wang M, Marotz C, Sprecher KE, Robles DV, Withrow D, Ackermann G, Herrera L, Bradford BJ, Marques LMM, Amaral JG, Silva RM, Veras FP, Cunha TM, Oliveira RDR, Louzada-Junior P, Mills RH, Galasko D, Dulai PS, Wittenberg C, Gonzalez DJ, Terkeltaub R, Doty MM, Kim JH, Rhee KE, Beauchamp-Walters J, Wright Jr KP, Dominguez-Bello MG, Manary M, Oliveira MF, Boland BS, Lopes NP, Guma M, Swafford AD, Dutton RJ, Knight R, Dorrestein PC. Reference data based insights expand understanding of human metabolomes. [_BioRxiv_](https://www.biorxiv.org/content/10.1101/2020.07.08.194159v1).
- Martino C, Kellman BP, Sandoval DR, Clausen TM, Marotz CA, Song SJ, Wandro S, Zaramela LS, Salido RA, **Zhu Q**, Armingol E, Vázquez-Baeza Y, McDonald D, Sorrentino JT, Taylor B, Belda-Ferre P, Liang C, Zhang Y, Schifanella L, Klatt NR, Havulinna AS, Jousilahti P, Huang S, Haiminen N, Parida L, Kim HC, Swafford AD, Zengler K, Cheng S, Inouye M, Niiranen T, Jain M, Salomaa V, Esko JD, Lewis NE, Knight R. Bacterial modification of the host glycosaminoglycan heparan sulfate modulates SARS-CoV-2 infectivity. [_BioRxiv_](https://www.biorxiv.org/content/10.1101/2020.08.17.238444v1).

## 2022

- **Zhu Q** (**co-corresponding**), Huang S, Gonzalez A, McGrath I, McDonald M, Haiminen N, Armstrong G, Vázquez-Baeza Y, **Yu J**, Kuczynski J, Sepich-Poore GD, Swafford AD, Das P, Shaffer JP, Lejzerowicz F, Belda-Ferre P, Havulinna AS, Méric G, Niiranen T, Lahti L, Salomaa V, Kim HC, Jain M, Inouye M, Gilbert JA, Knight R. Phylogeny-aware analysis of metagenome community ecology based on matched reference genomes while bypassing taxonomy. **_mSystems_**. In press. [_BioRxiv_](https://www.biorxiv.org/content/10.1101/2021.04.04.438427v1).
- Armstrong G, Martino C, Morris J, Khaleghi B, Kang J, Dereus J, **Zhu Q**, **Roush D**, McDonald D, Gonzalez A, Shaffer J, Carpenter C, Estaki M, Wandro S, Eilert S, Akel A, Eno J, Curewitz K, Swafford A, Moshiri N, Rosing T, Knight R. Swapping metagenomics preprocessing pipeline components offers speed and sensitivity increases. **_mSystems_**. In press.
- Singh K, Pituch P, **Zhu Q**, Gu H, Ernst B, Tofthagen C, Brewer B, Kober KM, Cooper BA, Paul SM, Conley YP, Hammer M, Levine JD, Miaskowski C. Distinct Nausea Profiles Are Associated With Gastrointestinal Symptoms in Oncology Patients Receiving Chemotherapy. [_Cancer Nursing_](https://journals.lww.com/cancernursingonline/Abstract/9000/Distinct_Nausea_Profiles_Are_Associated_With.98738.aspx). 2022. doi: 10.1097/NCC.0000000000001076.
- Needham BD, Funabashi M, Adame MD, Wang Z, Boktor JC, Haney J, Wu WL, Rabut C, Ladinsky MS, Hwang SJ, Guo Y, **Zhu Q**, Griffiths JA, Knight R, Bjorkman PJ, Shapiro MG, Geschwind DH, Holschneider DP, Fischbach MA, Mazmanian SK. A gut-derived metabolite alters brain activity and anxiety behavior in mice. [_Nature_](https://www.nature.com/articles/s41586-022-04396-8). 2022. 602(7898):647-53.
- Qin Y, Havulinna AS, Liu Y, Jousilahti P, Ritchie SC, Tokolyi A, Sanders JG, Valsta L, Brożyńska M, **Zhu Q**, Tripathi A, Vázquez-Baeza Y, Loomba R, Cheng S, Jain M, Niiranen T, Lahti L, Knight R, Salomaa V, Inouye M, Méric G. Combined effects of host genetics and diet on human gut microbiota and incident disease in a single population cohort. [_Nature Genetics_](https://www.nature.com/articles/s41588-021-00991-z). 2022. 54(2):134-42.
- Mills RH, Dulai PS, Vázquez-Baeza Y, Sauceda C, Daniel N, Gerner RR, Batachari LE, Malfavon M, **Zhu Q**, Weldon K, Humphrey G, Carrillo-Terrazas M, Goldasich LD, Bryant M, Raffatellu M, Quinn RA, Gewirtz AT, Chassaing B, Chu H, Sandborn WJ, Dorrestein PC, Knight R, Gonzalez DJ. Multi-omics analyses of the ulcerative colitis gut microbiome link _Bacteroides vulgatus_ proteases with disease severity. [_Nature Microbiology_](https://www.nature.com/articles/s41564-021-01050-3). 2022. 7(2):262-76.
- Jasbi P, Mohr AE, Shi X, Mahmood T, **Zhu Q**, Bruening M, Gu H, Whisner C. Microbiome and metabolome profiles of high screen time in a cohort of healthy college students. [_Scientific Reports_](https://www.nature.com/articles/s41598-022-07381-3). 2022. 12(1):1-17.
- Balaban M, Jiang Y, **Roush D**, **Zhu Q**, Mirarab S. Fast and Accurate Distance‐based Phylogenetic Placement using Divide and Conquer. [_Molecular Ecology Resources_](https://onlinelibrary.wiley.com/doi/abs/10.1111/1755-0998.13527). 2022. 22(3):1213-27.

## 2021

- **Zhu Q**, Hou Q, Huang S, Ou Q, Huo D, Vázquez-Baeza Y, Cen C, Cantu V, Estaki M, Chang H, Belda-Ferre P, Kim HC, Chen K, Knight R, Zhang J. Compositional and genetic alterations in Graves’ disease gut microbiome reveal specific diagnostic biomarkers. [_The ISME Journal_](https://www.nature.com/articles/s41396-021-01016-7). 2021. 15(11):3399-411.
- Armstrong G, Cantrell K, Huang S, McDonald D, Haiminen N, Carrieri AP, **Zhu Q**, Gonzalez A, McGrath I, Beck K, Hakim D, Havulinna AS, Méric G, Niiranen T, Lahti L, Salomaa V, Jain M, Inouye M, Swafford AD, Kim HC, Parida L, Vázquez-Baeza Y, Knight R. Efficient computation of Faith’s phylogenetic diversity with applications in characterizing microbiomes. [_Genome Research_](https://genome.cshlp.org/content/early/2021/09/03/gr.275777.121). 2021. 31(11):2131-37.
- Huang S, Jiang S, Huo D, Allaband C, Estaki M, Cantu V, Belda-Ferre P, Vázquez-Baeza Y, **Zhu Q**, Ma C, Li C, Zarrinpar A, Liu YY, Knight R, Zhang J. Candidate probiotic _Lactiplantibacillus plantarum_ HNU082 rapidly and convergently evolves within human, mice, and zebrafish gut but differentially influences the resident microbiome. [_Microbiome_](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01102-0). 2021. 9(1):151.
- Singh KP, Kober KM, Ernst B, Sachdev J, Brewer M, **Zhu Q**, Gu H, Melisko M, Paul SM, Cooper BA, Hammer M, Conley YP, Levine JD, Miaskowski C. Multiple Gastrointestinal Symptoms Are Associated With Chemotherapy-Induced Nausea In Patients With Breast Cancer. [_Cancer Nursing_](https://journals.lww.com/cancernursingonline/Abstract/9000/Multiple_Gastrointestinal_Symptoms_Are_Associated.98832.aspx). 2021. doi: 10.1097/NCC.0000000000000976.
- Sun Z, Huang S, Zhang M, **Zhu Q**, Haiminen M, Carrieri AP, Vázquez-Baeza Y, Parida L, Kim HC, Knight R, Liu YY. Challenges in Benchmarking Metagenomic Profilers. [_Nature Methods_](https://www.nature.com/articles/s41592-021-01141-3). 2021. 18(6):618-26.
- Jiao JY, Fu L, Hua ZS, Liu L, Salam N, Liu PF, Lv AP, Wu G, Xian WD, **Zhu Q**, Zhou EM, Fang BZ, Oren A, Hedlund BP, Jiang HC, Knight R, Cheng L, Li WJ. Insight into the function and evolution of Wood-Ljungdahl pathway in Actinobacteria. [_The ISME Journal_](https://www.nature.com/articles/s41396-021-00935-9). 2021. 15(10):3005-18.
- Karst SM, Ziels RM, Kirkegaard RH, Sørensen EA, McDonald D, **Zhu Q**, Knight R, Albertsen M. High-accuracy long-read amplicon sequences using unique molecular identifiers with Nanopore or PacBio sequencing. [_Nature Methods_](https://www.nature.com/articles/s41592-020-01041-y). 2021. 18(2):165-9.
- Tripathi A, Vazquez-Baeza Y, Gauglitz JM, Wang M, Duhrkop K, Esposito-Nothias M, Acharya DD, Ernst M, van der Hooft JJ, **Zhu Q**, McDonald D, Brejnrod AD, Gonzalez A, Handelsman J, Fleischauer M, Ludwig M, Böcker S, Nothias LF, Knight R, Dorrestein PC. Chemically informed analyses of metabolomics mass spectrometry data with Qemistree. [_Nature Chemical Biology_](https://www.nature.com/articles/s41589-020-00677-3). 2021. 17(2):146-51.
- Cantrell K, Fedarko MW, Rahman G, McDonald D, Yang Y, Zaw T, Gonzalez A, Janssen S, Estaki M, Haiminen N, Beck KL, **Zhu Q**, Sayyari E, Morton JT, Armstrong G, Tripathi A, Gauglitz JM, Marotz C, Matteson NL, Martino C, Sanders JG, Carrieri AP, Song SJ, Swafford AD, Dorrestein PC, Andersen KG, Parida L, Kim HC, Vázquez-Baeza Y, Knight R. EMPress Enables Tree-Guided, Interactive, and Exploratory Analyses of Multi-omic Data Sets. [_mSystems_](https://journals.asm.org/doi/10.1128/msystems.01216-20). 2021. 6(2):e01216-20.
- Bhagwat G, **Zhu Q**, O'Connor W, Subashchandrabose S, Grainge, I, Knight R, Palanisami T. Exploring the Composition and Functions of Plastic Microbiome Using Whole-Genome Sequencing. [_Environmental Science & Technology_](https://pubs.acs.org/doi/abs/10.1021/acs.est.0c07952). 2021. 55(8):4899-913.
- Leão T, Wang M, Moss N, da Silva R, Sanders J, Nurk S, Gurevich A, Humphrey G, Reher R, **Zhu Q**, Belda-Ferre P, Glukhov E, Whitner S, Alexander KL, Rex R, Pevzner P, Dorrestein PC, Knight R, Bandeira N, Gerwick WH, Gerwick L. A Multi-Omics Characterization of the Natural Product Potential of Tropical Filamentous Marine Cyanobacteria. [_Marine Drugs_](https://www.mdpi.com/1660-3397/19/1/20). 2021. 19(1):20.

## 2020

- Lin W, Zhang W, Paterson GA, **Zhu Q**, Zhao X, Knight R, Bazylinski DA, Roberts AP, Pan Y. Expanding magnetic organelle biogenesis in the domain Bacteria. [_Microbiome_](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00931-9). 2020. 8(1):152.
- Asnicar F, Thomas AM, Beghini F, Mengoni C, Manara S, Manghi P, **Zhu Q**, Bolzan M, Cumbo F, May U, Sanders JG, Zolfo M, Kopylova E, Pasolli E, Knight R, Mirarab S, Huttenhower C, Segata N. Precise phylogenetic analysis of microbial isolates and genomes from metagenomes using PhyloPhlAn 3.0. [_Nature Communications_](https://www.nature.com/articles/s41467-020-16366-7). 2020. 11(1):2500.
- Hillmann B, Al-Ghalith GA, Shields-Cutler RR, **Zhu Q**, Knight R, Knights D. SHOGUN: a modular, accurate, and scalable framework for microbiome quantification. [_Bioinformatics_](https://academic.oup.com/bioinformatics/article-abstract/36/13/4088/5828930). 2020. 36(13):4088-90.
- Estaki M, Jiang L, Bokulich NA, McDonald D, González A, Kosciolek T, Martino C, **Zhu Q**, Birmingham A, Vázquez-Baeza Y, Dillon MR, Bolyen E, Caporaso JG, Knight R. QIIME 2 enables comprehensive end-to-end analysis of diverse microbiome data and comparative studies with publicly available data. [_Current Protocols in Bioinformatics_](https://currentprotocols.onlinelibrary.wiley.com/doi/full/10.1002/cpbi.100). 2020. 70(1):e100.
- Poore GD, Kopylova E, **Zhu Q**, Carpenter C, Fraraccio S, Wandro S, Kosciolek T, Janssen S, Metcalf J, Song SJ, Kanbar J, Miller-Montgomery S, Heaton R, Mckay R, Patel SP, Swafford AD, Patel SP, Knight R. Microbiome analyses of blood and tissues suggest cancer diagnostic approach. [_Nature_](https://www.nature.com/articles/s41586-020-2095-1). 2020. 579(7800):567-74.
- McCall L, Callewaert C, **Zhu Q** (**co-first**), Song SJ, Bouslimani A, Minich JJ, Ernst M, Ruiz-Calderon JF, Cavallin H, Pereira HS, Novoselac A, Hernandez J, Rios R, Branch OH, Blaser MJ, Paulino LC, Dorrestein PC, Knight R, Dominguez-Bello MG. Home chemical and microbial transitions across urbanization. [_Nature Microbiology_](https://www.nature.com/articles/s41564-019-0593-4). 2020. 5(1):108-15.

## 2019

- **Zhu Q**, Mai U, Pfeiffer W, Janssen S, Asnicar F, Sanders JG, Belda-Ferre P, Al-Ghalith GA, Kopylova E, McDonald D, Kosciolek T, Yin JB, Huang S, Salam N, Jiao J, Wu Z, Xu ZZ, Sayyari E, Morton JT, Podell S, Knights D, Li W, Huttenhower C, Segata N, Smarr L, Mirarab S, Knight R. Phylogenomics of 10,575 genomes reveals evolutionary proximity between domains Bacteria and Archaea. [_Nature Communications_](https://www.nature.com/articles/s41467-019-13443-4). 2019. 10(1):5477.
- Sanders JG, Nurk S, Salido RA, Minich J, Xu ZZ, **Zhu Q**, Martino C, Fedarko M, Arthur TD, Chen F, Boland BS, Humphrey GC, Brennan C, Sanders K, Gaffney J, Jepsen K, Khosroheidari M, Green C, Liyanage M, Dang JW, Phelan VV, Quinn RA, Bankevich A, Chang JT, Rana TM, Conrad DJ, Sandborn WJ, Smarr L, Dorrestein PC, Pevzner PA, Knight R. Optimizing sequencing protocols for leaderboard metagenomics by combining long and short reads. [_Genome Biology_](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1834-9). 2019. 20(1):226.
- Bolyen E, Rideout JR, Dillon MR, Bokulich NA, Abnet CC, Al-Ghalith GA, Alexander H, Alm EJ, Arumugam M, Asnicar F, Bai Y, Bisanz JE, Bittinger K, Brejnrod A, Brislawn CJ, Brown CT, Callahan BJ, Caraballo-Rodríguez AM, Chase J, Cope EK, Da Silva R, Diener C, Dorrestein PC, Douglas GM, Durall DM, Duvallet C, Edwardson CF, Ernst M, Estaki M, Fouquier J, Gauglitz JM, Gibbons SM, Gibson DL, Gonzalez A, Gorlick K, Guo J, Hillmann B, Holmes S, Holste H, Huttenhower C, Huttley GA, Janssen S, Jarmusch AK, Jiang L, Kaehler BD, Kang KB, Keefe CR, Keim P, Kelley ST, Knights D, Koester I, Kosciolek T, Kreps J, Langille MGI, Lee J, Ley R, Liu YX, Loftfield E, Lozupone C, Maher M, Marotz C, Martin BD, McDonald D, McIver LJ, Melnik AV, Metcalf JL, Morgan SC, Morton JT, Naimey AT, Navas-Molina JA, Nothias LF, Orchanian SB, Pearson T, Peoples SL, Petras D, Preuss ML, Pruesse E, Rasmussen LB, Rivers A, Robeson MS nd, Rosenthal P, Segata N, Shaffer M, Shiffer A, Sinha R, Song SJ, Spear JR, Swafford AD, Thompson LR, Torres PJ, Trinh P, Tripathi A, Turnbaugh PJ, Ul-Hasan S, van der Hooft JJJ, Vargas F, Vázquez-Baeza Y, Vogtmann E, von Hippel M, Walters W, Wan Y, Wang M, Warren J, Weber KC, Williamson CHD, Willis AD, Xu ZZ, Zaneveld JR, Zhang Y, **Zhu Q**, Knight R, Caporaso JG. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. [_Nature Biotechnology_](https://www.nature.com/articles/s41587-019-0209-9). 2019. 37(8):852-57.
- Fu T, Coulter S, Yoshihara E, Oh TG, Fang S, Cayabyab F, **Zhu Q**, Zhang T, Leblanc M, Liu S, He M, Waizenegger W, Gasser E, Schnabl B, Atkins AR, Yu RT, Knight R, Liddle C, Downes M, Evans RM. FXR Regulates Intestinal Cancer Stem Cell Proliferation. [_Cell_](https://www.cell.com/cell/fulltext/S0092-8674(19)30099-6). 2019. 176(5):1098-1112.e18.
- Mills RH, Vázquez-Baeza Y, **Zhu Q**, Jiang L, Gaffney J, Humphrey G, Smarr L, Knight R, Gonzalez DJ. Evaluating Metagenomic Prediction of the Metaproteome in a 4.5-Year Study of a Patient with Crohn's Disease. [_mSystems_](https://journals.asm.org/doi/10.1128/msystems.00337-18). 2019. 4(1):e00337-18.
- Xu ZZ, Amir A, Sanders J, **Zhu Q**, Morton JT, Bletz MC, Tripathi A, Huang S, McDonald D, Jiang L, Knight R. Calour: an Interactive, Microbe-Centric Analysis Tool. [_mSystems_](https://journals.asm.org/doi/10.1128/msystems.00269-18). 2019. 4(1):e00269-18.

## 2018

- **Zhu Q**, Dupont CL, Jones MB, Pham KM, Jiang ZD, DuPont HL, Highlander SK. Visualization-assisted binning of metagenome assemblies reveals potential new pathogenic profiles in idiopathic travelers' diarrhea. [_Microbiome_](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0579-0). 2018. 6(1):201.
- Fang X, Monk J, Nurk S, Akseshina M, **Zhu Q**, Gemmell C, Gianetto-Hill C, Leung N, Szubin R, Sanders J, Beck PL, Li W, Sandborn WJ, Gray-Owen SD, Knight R, Allen-Vercoe E, Palsson BO, Smarr L. Metagenomics-based, strain-level analysis of Escherichia coli from a time-series of microbiome samples from a Crohn’s disease patient. [_Frontiers in Microbiology_](https://www.frontiersin.org/articles/10.3389/fmicb.2018.02559/full). 2018. 9:2559.
- Hillmann B, Al-Ghalith G, Shields-Cutler R, **Zhu Q**, Gohl D, Beckman K, Knight R, Knights D. Evaluating the information content of shallow shotgun metagenomics. [_mSystems_](https://msystems.asm.org/content/3/6/e00069-18). 2018. 3(6):e00069-18.
- Minich JJ, **Zhu Q**, Xu ZZ, Amir A, Ngochera M, Simwaka M, Allen EE, Zidana H, Knight R. Microbial effects of livestock manure fertilization on freshwater aquaculture ponds rearing tilapia (_Oreochromis shiranus_) and North African catfish (_Clarias gariepinus_). [_MicrobiologyOpen_](https://onlinelibrary.wiley.com/doi/full/10.1002/mbo3.716). 2018. e00716.
- Tripathi A, Marotz C, Gonzalez A, Vázquez-Baeza Y, Song SJ, Bouslimani A, McDonald D, **Zhu Q**, Sanders JG, Smarr L, Dorrestein PC, Knight R. Are microbiome studies ready for hypothesis-driven research? [_Current Opinion in Microbiology_](https://www.sciencedirect.com/science/article/abs/pii/S1369527418300511). 2018. 44:61-69.
- Hua ZS, Qu YN, **Zhu Q**, Zhou EM, Qi YL, Yin YR, Rao YZ, Tian Y, Li YX, Liu L, Castelle CJ, Hedlund BP, Shu WS, Knight R, Li WJ. Genomic inference of the metabolism and evolution of the archaeal phylum Aigarchaeota. [_Nature Communications_](https://www.nature.com/articles/s41467-018-05284-4). 2018. 9(1):2832.
- McDonald D, Hyde E, Debelius JW, Morton JT, Gonzalez A, Ackermann G, Aksenov AA, Behsaz B, Brennan C, Chen Y, Goldasich LD, Dorrestein PC, Dunn RR, Fahimipour AK, Gaffney J, Gilbert JA, Gogul G, Green JL, Hugenholtz P, Humphrey G, Huttenhower C, Jackson MA, Janssen S, Jeste DV, Jiang L, Kelley ST, Knights D, Kosciolek T, Ladau J, Leach J, Marotz C, Meleshko D, Melnik AV, Metcalf JL, Mohimani H, Montassier E, Navas-Molina J, Nguyen TT, Peddada S, Pevzner P, Pollard KS, Rahnavard G, Robbins-Pianka A, Sangwan N, Shorenstein J, Smarr L, Song SJ, Spector T, Swafford AD, Thackray VG, Thompson LR, Tripathi A, Vázquez-Baeza Y, Vrbanac A, Wischmeyer P, Wolfe E, **Zhu Q**, The American Gut Consortium, Knight R. American Gut: an Open Platform for Citizen Science Microbiome Research. [_mSystems_](https://journals.asm.org/doi/10.1128/msystems.00031-18). 3(3):e00031-18.
- He Y, Kosciolek T, Tang J, Zhou Y, Li Z, Ma X, **Zhu Q**, Yuan N, Yuan L, Li C, Jin K, Knight R, Tsuang MT, Chen X. Gut microbiome and magnetic resonance spectroscopy study of subjects at ultra-high risk for psychosis may support the membrane hypothesis. [_European Psychiatry_](https://www.sciencedirect.com/science/article/abs/pii/S0924933818301068). 2018. 53:37-45.
- Washburne AD, Morton JT, Sanders J, McDonald D, **Zhu Q**, Oliverio AM, Knight R. Methods for phylogenetic analysis of microbiome data. [_Nature Microbiology_](https://www.nature.com/articles/s41564-018-0156-0). 2018. 3(6):652.
- Knight R, Vrbanac A, Taylor BC, Aksenov A, Callewaert C, Debelius J, Gonzalez A, Kosciolek T, McCall LI, McDonald D, Melnik AV, Morton JT, Navas J, Quinn RA, Sanders JG, Swafford AD, Thompson LR, Tripathi A, Xu ZZ, Zaneveld JR, **Zhu Q**, Caporaso JG, Dorrestein PC. Best practices for analysing microbiomes. [_Nature Reviews Microbiology_](https://www.nature.com/articles/s41579-018-0029-9). 2018. 16(7):410-422.
- Minich JJ, **Zhu Q**, Janssen S, Hendrickson R, Amir A, Vetter R, Hyde J, Doty MM, Stillwell K, Benardini J, Kim JH, Allen EE, Venkateswaran K, Knight R. KatharoSeq Enables High-Throughput Microbiome Analysis from Low-Biomass Samples. [_mSystems_](https://journals.asm.org/doi/10.1128/msystems.00218-17). 2018. 3(3):e00218-17.
- Kapono CA, Morton JT, Bouslimani A, Melnik AV, Orlinsky K, Knaan TL, Garg N, Vázquez-Baeza Y, Protsyuk I, Janssen S, **Zhu Q**, Alexandrov T, Smarr L, Knight R, Dorrestein PC. Creating a 3D microbial and chemical snapshot of a human habitat. [_Scientific Reports_](https://www.nature.com/articles/s41598-018-21541-4). 2018. 8(1):3669.

## 2017 and before

- Thompson LR, Sanders JG, McDonald D, Amir A, Ladau J, Locey KJ, Prill RJ, Tripathi A, Gibbons SM, Ackermann G, Navas-Molina JA, Janssen S, Kopylova E, Vázquez-Baeza Y, González A, Morton JT, Mirarab S, Zech Xu Z, Jiang L, Haroon MF, Kanbar J, **Zhu Q**, Jin Song S, Kosciolek T, Bokulich NA, Lefler J, Brislawn CJ, Humphrey G, Owens SM, Hampton-Marcell J, Berg-Lyons D, McKenzie V, Fierer N, Fuhrman JA, Clauset A, Stevens RL, Shade A, Pollard KS, Goodwin KD, Jansson JK, Gilbert JA, Knight R, The Earth Microbiome Project Consortium. A communal catalogue reveals Earth’s multiscale microbial diversity. [_Nature_](https://www.nature.com/articles/nature24621). 2017. 551(7681):457-63.
- Lin W, Paterson GA, **Zhu Q**, Wang Y, Kopylov E, Li Y, Knight R, Bazylinski DA, Zhu R, Kirschvink JL, Pan Y. Origin of microbial biomineralization and magnetotaxis during the Archean. [_Proceedings of the National Academy of Sciences_](https://www.pnas.org/content/114/9/2171). 2017. 114(9):2171-76.
- Yang Y, **Zhu Q**, Liu S, Zhao C, Wu C. The origin of Chinese domestic horses revealed with novel mtDNA variants. [_Animal Science Journal_](https://www.onlinelibrary.wiley.com/doi/abs/10.1111/asj.12583). 2017. 88(1):19-26.
- Li W, Richter RA, Jung Y, **Zhu Q**, Li RW. Web-based bioinformatics workflows for end-to-end RNA-seq data computation and analysis in agricultural animal species. [_BMC Genomics_](https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-3118-z). 2016. 17(1):761.
- Dittmar K, **Zhu Q**, Hastriter MW, Whiting MF: On the probability of dinosaur fleas. [_BMC Evolutionary Biology_](https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-015-0568-x). 2016. 16(1):9.
- Liu Y, Yu Y, Duan W, Qu Q, Zhang Q, Zhao M, **Zhu Q**. Home storage significantly impairs Bifidobacteria survival in powered formula for infants and young children in the Chinese market. [_International Journal of Dairy Technology_](https://www.onlinelibrary.wiley.com/doi/10.1111/1471-0307.12262). 2015. 68(4):495-502.
- Han X, Chen Y, Wang L, Fang W, Zhang N, **Zhu Q**. RNA: an expanding view of function and evolution. [_Evolutionary Bioinformatics_](https://journals.sagepub.com/doi/full/10.4137/EBO.S38105). 2015. Suppl. 1:77-9.
- **Zhu Q**, Hastriter MW, Whiting MF, Dittmar K. Fleas (Siphonaptera) are Cretaceous, and evolved with Theria. [_Molecular Phylogenetics and Evolution_](https://www.sciencedirect.com/science/article/abs/pii/S1055790315001293). 2015. 90:129-39.
- Brook CE, Bai Y, Dobson AP, Osikowicz LM, Ranaivoson HC, **Zhu Q**, Kosoy MY, Dittmar K. _Bartonella_ spp. in fruit bats and blood-feeding ectoparasites in Madagascar. [_PLOS Neglected Tropical Diseases_](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0003532). 2015. 9(2):e0003532.
- **Zhu Q** (**co-corresponding**), Kosoy M, Dittmar K. HGTector: an automated method facilitating genome-wide discovery of putative horizontal gene transfers. [_BMC Genomics_](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-717). 2014. 15:717.
- **Zhu Q** (**co-corresponding**), Kosoy M, Olival KJ, Dittmar K. Horizontal transfers and gene losses in the phospholipid pathway of _Bartonella_ reveal clues about early ecological niches. [_Genome Biology and Evolution_](https://academic.oup.com/gbe/article/6/8/2156/2939846). 2014. 6(8):2156-69.
- Kazemian M, **Zhu Q**, Halfon MS, Sinha S. Improved accuracy of supervised CRM discovery with interpolated Markov models and cross-species comparison. [_Nucleic Acids Research_](https://academic.oup.com/nar/article/39/22/9463/2409012). 2011. 39(22):9463-72.
- Kantorovitz MR, Kazemian M, Kinston S, Miranda-Saavedra D, **Zhu Q**, Robinson GE, Göttgens B, Halfon MS, Sinha S. Motif-blind, genome-wide discovery of _cis_-regulatory modules in _Drosophila_ and mouse. [_Developmental Cell_](https://www.sciencedirect.com/science/article/pii/S1534580709003840). 2009. 17(4):568-79.
- Liu Y, **Zhu Q** (**co-first**), Zhu N. Recent duplication and positive selection of the _GAGE_ gene family. [_Genetica_](https://link.springer.com/article/10.1007/s10709-007-9179-9). 2008. 133(1):31-5.
- Liu Y, **Zhu Q**, Zhu N. Rapid HLA-DR fluorotyping based on melting curve analysis. [_Immunological Investigations_](https://www.tandfonline.com/doi/full/10.1080/08820130701366425). 2007. 36(4):507-21.

## Book chapters

- Dittmar K, **Zhu Q**, Hastriter MW, Whiting MF. Evolutionary history of Siphonaptera: fossils, origins, vectors. In [_Parasite Diversity and Diversification - Evolutionary Ecology Meets Phylogenetics_](https://www.amazon.com/Parasite-Diversity-Diversification-Evolutionary-Phylogenetics/dp/1107037654). Ed. Morand S, Krasnov BR, Littlewood DTJ. Cambridge University Press. 2015.
